CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 3.13.4)
+
+project(SuperTux VERSION 0.6.3 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# This should be set only with windows Visual Studio generator builds
+string(TOLOWER "${CMAKE_GENERATOR_PLATFORM}" PLATFORM)
+if(${PLATFORM} MATCHES "arm64")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /D_ARM64_ /DMY_CPU_LE")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /D_ARM64_ /DMY_CPU_LE")
+endif()
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/mk/cmake)
+include(ConfigureFiles)
+include(ExternalProject)
+
+## For autopackage
+set(APPDATADIR "${CMAKE_INSTALL_PREFIX}/share/games/supertux2")
+
+set(BUILD_DATA_DIR "${PROJECT_SOURCE_DIR}/data")
+set(BUILD_CONFIG_DATA_DIR "${PROJECT_BINARY_DIR}/data")
+
+## Add definitions
+if(CMAKE_BUILD_TYPE MATCHES Rel)
+  option(IS_SUPERTUX_RELEASE "Build as official SuperTux release" ON)
+  add_definitions(-DRELEASE)
+else()
+  option(IS_SUPERTUX_RELEASE "Build as official SuperTux release" OFF)
+  add_definitions(-DDEBUG)
+endif()
```

Comment:
Maybe instead of duplicating `option`, use helper variable like `IS_SUPERTUX_RELEASE_DEFAULT` and then pass that to `option`.